cmd/powd/main.go

```diff
@@ -276,12 +269,35 @@ func setupLogging(repoPath string) error {
 		"user-service",
 	}
 
-	// powd registered loggers get info level by default.
-	for _, l := range loggers {
-		if err := logging.SetLogLevel(l, "info"); err != nil {
-			return fmt.Errorf("setting up logger %s: %s", l, err)
+	var ipfslog bool
+	// Looking for ipfs/go-log setup environment variables
+	// If at least one of them defined - do not override
+	// ipfs/go-log internal logging setup
+	for _, e := range os.Environ() {
+		if strings.HasPrefix(e, "GOLOG_") {
+			ipfslog = true
+			break
+		}
+	}
+	if !ipfslog {
+		if err := os.MkdirAll(repoPath, os.ModePerm); err != nil {
+			return fmt.Errorf("creating repo folder: %s", err)
+		}
+		cfg := logging.Config{
+			Level:  logging.LevelError,
+			Stdout: true,
+			File:   filepath.Join(repoPath, "powd.log"),
+		}
+		logging.SetupLogging(cfg)
+
+		// powd registered loggers get info level by default.
+		for _, l := range loggers {
+			if err := logging.SetLogLevel(l, "info"); err != nil {
+				return fmt.Errorf("setting up logger %s: %s", l, err)
+			}
```

Comment:
Merged master in, not sure why this is showing up, but let me know if it looks wrong.